docker-compose-schema-validator.yml

```diff
@@ -1,13 +1,13 @@
 services:
   ajv-validator:
-    image: onsdigital/eq-questionnaire-validator:${TAG}-ajv
```

Comment:
I think this `TAG` is being used by `./scripts/run_validator.sh`, which is called from a few different places, so changing this to always be `latest` might have an impact in other places, like with `make run-validator`

I'm guessing this file was never intended to be called directly without that wrapper script, so I'd be inclined to keep the `TAG` in for flexibility for now?